DebugView++/MainFrame.cpp

```diff
@@ -268,7 +268,7 @@ std::wstring FormatDuration(double seconds)
 	if (minutes > 0)
 		return wstringbuilder() << FormatUnits(minutes, L"minute") << L" " << FormatUnits(floor_to<int>(seconds), L"second");
 
-	static const wchar_t* units[] = { L"s", L"ms", L"�s", L"ns", nullptr };
+	static const wchar_t* units[] = { L"s", L"ms", L"µs", L"ns", nullptr };

```

Comment:
Hmm, the encoding of µ changed to UTF-8. Perhaps this should be reverted. Sorry.